tests/inspect/inspect.go

```diff
@@ -97,6 +101,15 @@ func init() {
 	globalFlagset.IntVar(&globalFlags.ServeHttpTimeout, "serve-http-timeout", 30, "HTTP Timeout to wait for a client connection")
 }
 
+func in(list []int, el int) bool {
```

Comment:
Wooo! Another implementation of `in` operator. I'll have to start adding some checks everytime I see it. :)

Disregard this comment. ;)